documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_1_Security_Perspective/4_1_6_Usage_Control.md

```diff
@@ -1 +1,263 @@
+
+The management of usage policies becomes especially important when data is to be exchanged across system boundaries. Every time data crosses system boundaries, the target system must be prepared for the protection of incoming data (i.e. it has to deploy the corresponding policy).
+
+#### **USAGE CONTROL IN A CONNECTOR**
+
+Usage Control only makes sense in an ecosystem where a certain level of trust can be established and maintained for all participants. To enable the establishment of trusted relationships, the central technological components used for data processing and data exchange need to be trustworthy. The IDS Connector is the central component for data exchange and data processing in the International Data Spaces, and thus a central component that needs to be trusted.
+
+The IDS Connector focuses on security and delivers a trusted platform, incorporating crucial building blocks:
+
+» identity & trust management for authenticating communicating parties (e.g., other Connectors) and shaping trusted relationships between partners;
+
+» a trusted platform as a baseline for secure data processing;
+
+» trustworthy communication based on authenticated and encrypted connections; and
+
+» Access & Usage Control.
+
+Instances of the Trusted Connector enable remote integrity verification, so the integrity of the deployed software stack can be guaranteed before granting access to data.
+
+The Trusted Connector guarantees a controlled execution environment for data services and supports the creation of trusted relationships. A general constraint is one that remains for all deployed IT systems: As long as physical or logical access is granted to administrators, protection against data theft by malicious partners is almost impossible to prevent. The International Data Spaces is seen as a network of partners that are provided with the technical means to fulfill their obligations and support in deciding what partners to trust and to define reasonable access conditions.
+
+#### **APACHE CAMEL INTERCEPTOR (EXAMPLE)**
+
+An IDS Connector may use Apache Camel to coordinate the data flow between different systems and applications. From a technical point of view, the developer does this by using pipelining, which is a dominant paradigm of Apache Camel for connecting different nodes in a route definition. The basic idea of a pipeline is that Apache Camel uses the output of one node as input to the next node. Every node in such a route is a processor, except for the initial endpoint (as shown in [Figure 4.1.6.6](https://user-images.githubusercontent.com/69632955/150939403-107db526-d0a6-47f4-85d3-30a2b29e4f9b.png)).
+
+![image](https://user-images.githubusercontent.com/69632955/150939403-107db526-d0a6-47f4-85d3-30a2b29e4f9b.png)

```

Comment:
This should be PEP or Policy Engine instead of LUCON (which is an implementation of an Policy Engine)

Reply:
this is an example of differnt engines both using usally a PEPs. But we can change the picture to PEPs or Policy Engines